### PR TITLE
`Rails/FindEach` cop works with `where.not`.

### PR DESCRIPTION
**Pull request by pocke**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 * Fix auto-correction of `not` with parentheses in `Style/Not`. ([@lumeet][])
 
+### New features
```
Comment (bbatsov) [inline CHANGELOG.md]: We typically place this section above `Bug Fixes`.

**Comment by bbatsov** [comment]
Please, drop the `.` from the commit message.

**Comment by pocke** [comment]
I see.
I'll fix commit message and section in CHANGELOG.  And I'll push it.

**Comment by pocke** [comment]
Thanks!

**Pull request merged**